### PR TITLE
feat: Authorization

### LINKED ISSUE #34: [Feat]: Auth
### My suggestion

Auth using email + password and/or google

### Possible alternatives

_No response_

### PR DESCRIPTION
**Pull request by Norbiros**
Resolves #34

### CODE REVIEW

**Review by Kubaryt — CHANGES_REQUESTED** [review]
Add registration

**Comment by Norbiros** [comment]
Note: UI is temporary

File: backend/routers/index_router.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ async def read_main():
     return {"msg": "Hello World"}
 
 
-@index_router.get("/get_users", response_model=List[schemas.User])
-def get_users(db: Session = Depends(get_db)):
-    users = db.query(models.User).all()
+@index_router.get("/get_pixel_watch", response_model=List[schemas.PixelWatch])
```
Comment (wikipop) [inline backend/routers/index_router.py:26]: Imo this should be in different router

**Review by wikipop — APPROVED** [review]

File: backend/routers/index_router.py (hunk 1 above)
Comment (Norbiros) [reply backend/routers/index_router.py:26]: Hmm, maybe I should just remove this route?

Comment (wikipop) [reply backend/routers/index_router.py:26]: Whatever idk what it has to to do with this pr

**Review by wikipop — APPROVED** [review]

**Pull request merged**